### PR TITLE
[W5.4][T16-A2]Kwek Jun Kai Sean

### PR DESCRIPTION
**Pull request by seankwekjk**
add case-insensitive functionality to the find command
update IO test cases and JUnit tests to reflect

### CODE REVIEW

**Comment by Lenaldnwj** [comment]
Hi Sean, I feel that you should provide a description above the 

public List<String> getWordsInNameLower(){
        String name=fullName.toLowerCase();
        return Arrays.asList(name.split("\\s+"));
    }

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -43,6 +43,11 @@ public static boolean isValidName(String test) {
         return Arrays.asList(fullName.split("\\s+"));
     }
 
+    public List<String> getWordsInNameLower(){
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Name.java]: Missing header comment. All non trivial methods should have javadoc format header comments.

File: src/seedu/addressbook/data/person/Name.java [hunk 2]
```diff
@@ -43,6 +43,11 @@ public static boolean isValidName(String test) {
...
+        String name=fullName.toLowerCase();
+        return Arrays.asList(name.split("\\s+"));
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Name.java]: Perhaps `"\\s+"` can be refactored out, this helps with code readability.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 3]
```diff
@@ -17,14 +17,18 @@
...
+            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " alice bob charlie";
 
     private final Set<String> keywords;
 
     public FindCommand(Set<String> keywords) {
-        this.keywords = keywords;
+        Set<String> finalKeywords=new HashSet<>();
+        for(String string : keywords){
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/commands/FindCommand.java]: [Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. Missing space between open brace

File: src/seedu/addressbook/data/person/Name.java [hunk 4]
```diff
@@ -43,6 +43,11 @@ public static boolean isValidName(String test) {
         return Arrays.asList(fullName.split("\\s+"));
     }
 
+    public List<String> getWordsInNameLower(){
+        String name=fullName.toLowerCase();
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Name.java]: [Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. extra spaces.

**Pull request closed without merging**